lib/decision_review_v1/utilities/helpers.rb

```diff
@@ -29,12 +29,15 @@ def get_and_rejigger_required_info(request_body:, form4142:, user:)
           "vaFileNumber": user.ssn.to_s.strip.presence,
           "veteranSocialSecurityNumber": user.ssn.to_s.strip.presence,
           "veteranFullName": {
-            "first": user.first_name.to_s.strip.first(12),
+            "first": user.first_name.to_s,
             "middle": middle_initial(user),
-            "last": user.last_name.to_s.strip.first(18).presence
+            "last": user.last_name.to_s.presence
```

Comment:
Removed name formatting as that process is preformed by the `MetadataValidator` when generating the metadata object.